lib/commands/alert.js

```diff
@@ -138,12 +155,16 @@ helpers.getAlert = async function () {
   }
 
   alert.getText = async () => {
-    let textView = await this.findNativeElementOrElements('class name', 'XCUIElementTypeTextView', false, util.unwrapElement(alert));
+    let textView = await this.findNativeElementOrElements(
```

Comment:
Did you mean to commit this?
I think it is the same?

Reply:
Just keeping it below 100 chars.